`docker-compose up -d` throws error `Cannot find module '/usr/lib/node_modules/npm/lib/build.js'`

I'm trying to follow the [Setting up Habitica Locally](http://habitica.wikia.com/wiki/Setting_up_Habitica_Locally), but when I get to the step of running `docker-compose up -d`, dozens of the following errors appear in the console and the process exits with a non-zero code:

```
npm ERR! Cannot find module '/usr/lib/node_modules/npm/lib/build.js'
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Linux 4.4.12-boot2docker
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "gulp" "grunt-cli" "bower" "npm@3"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code MODULE_NOT_FOUND
```

It appears that this is an issue with the upgrade to `npm v3` coming after other `npm install`'s. If remove `npm@3` from [this line](https://github.com/HabitRPG/habitrpg/blob/develop/Dockerfile#L28) and add a separate call before it to upgrade npm everything works fine.

Let me know if you would like me to submit a pull request. I haven't been able to find any other information on this issue.

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,7 +25,8 @@ RUN apt-get clean
 RUN rm -rf /var/lib/apt/lists/*
 
 # Install global packages
-RUN npm install -g gulp grunt-cli bower npm@3
+RUN npm install -g npm@3
+RUN npm install -g gulp grunt-cli bower
 
 # Clone Habitica repo and install dependencies
 WORKDIR /habitrpg

